trainee_template/train.amlg

```diff
@@ -236,7 +245,7 @@
 				encode_features_on_train (false)
 			))
 		)
-
+		
```

Comment:
Just commenting a sorta random spot in Train. 

Should something be added in train that sets #hasInfluenceWeightEntropies to False?

Reply:
It's set to False in `#InitializeValues`